### PR TITLE
Rename pull-request-triage.md to pull-requests.md

### PR DESCRIPTION
**Pull request by outofambit**
this has been confusing people as we onboard and kind of irks me so here's a quick fix.

(also removed some snark along the way)

### CODE REVIEW

**Review by mislav — DISMISSED** [review]
Thanks; makes sense! ✨

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ to see if the enhancement has already been suggested. If it has, add a
 :thumbsup: to indicate your interest in it, or comment if there is additional
 information you would like to add.
 
-#### How Do I Submit A (Good) Enhancement Suggestion?
+#### How Do I Submit A Enhancement Suggestion?
```
Comment (billygriffin) [inline .github/CONTRIBUTING.md]:
```suggestion
#### How Do I Submit An Enhancement Suggestion?
```

**Review by billygriffin — CHANGES_REQUESTED** [review]
Super minor suggestion.

**Review by billygriffin — APPROVED** [review]

**Comment by outofambit** [comment]
merging! :)

**Pull request merged**